franklin-cloud/provision/franklin.sh

```diff
@@ -0,0 +1,46 @@
+#
+# Global Installs
+#
+
+sudo apt-get update
+sudo apt-get -y install apt-transport-https
+sudo apt-get -y install vim
+
+#
+# Create a 'Franklin' user, with the password 'franklin'
+#
+useradd -m -p pa4go24cAi1hA -s /bin/bash franklin
```

Comment:
franklin user with a button mash password? Should this be setup question asked to the installing user?